Readme.rst

```diff
@@ -59,6 +59,8 @@ To run tests, run ``./bin/phpunit -c ./Build/BuildEssentials/PhpUnit/UnitTests.x
 To switch the branch you intend to work on:
 ``git checkout 8.0 && composer update``
 
-.. note:: We use an upmerging strategy, so create all bugfixes to lowest maintained branch that contains the issue (typically the second last LTS release, which is 5.3 currently), or master for new features.
+.. note:: We use an upmerging strategy, so create all bugfixes to lowest maintained branch that
+contains the issue (typically the second last LTS release, check the diagram on
+https://www.neos.io/features/release-process.html), or master for new features.
```

Comment:
`master` does not exist anymore. It should say something like `the next minor or major version branch`, but a bit clearer.